src/csharpExtensionExports.ts

```diff
@@ -20,4 +22,10 @@ export interface CSharpExtensionExports {
     initializationFinished: () => Promise<void>;
     logDirectory: string;
     profferBrokeredServices: (container: GlobalBrokeredServiceContainer) => void;
+    sendRequest: <Params, Response, Error>(
```

Comment:
```suggestion
    sendServerRequest: <Params, Response, Error>(
```

Or maybe something a bit more verbose, if only because I can imagine we might also have other places to send requests too? (the Razor LSP says hi...)